src/listeners/decorators/once.decorator.ts

```diff
@@ -0,0 +1,13 @@
+import { Listener } from 'necord';
+import { LavalinkManagerEvents } from 'lavalink-client/dist/types/structures/Types/Manager';
+import { NodeManagerEvents } from 'lavalink-client';
+
+/**
+ * Decorator that marks a method as a listener for Lavalink client.
+ * @param event The event name.
+ * @returns The decorated method.
+ * @url https://necord.org/recipes/lavalink/listeners
+ */
+export const OnceLavalink = <K extends keyof E, E = LavalinkManagerEvents | NodeManagerEvents>(
+	event: K
+) => Listener({ type: 'once', event });
```

Comment:
Can we here `Once` ? It also have generic types

Reply:
Sorry I didn't understand the question

Reply:
now I realized that in this decorator I am not using the NecordLavalinkEvents interface

Reply:
From the discussion below, we can replace decorators from necord by own

Reply:
Could you show me an example?